src/sonic-bgpcfgd/bgpcfgd/managers_bbr.py

```diff
@@ -106,15 +107,18 @@ def __set_prepare_config(self, status):
         :return: list of commands prepared for FRR
         """
         bgp_asn = self.directory.get_slot("CONFIG_DB", swsscommon.CFG_DEVICE_METADATA_TABLE_NAME)["localhost"]["bgp_asn"]
+        self.cfg_mgr.update()
```

Comment:
@pavel-shirshov what is the use case of update() here ?

Reply:
To download latest 'show run' from frr.
Otherwise it would be None.

P.S.: It is better to have this call in the runner, to update the configuration at the start of update cycle. and then we need to check both 'show run' records and proposed records changes, in case managers works with configuration which is changed in many places.